packages/contracts/package.json

```diff
@@ -9,7 +9,7 @@
     "clean": "forge clean && rm -rf broadcast cache deployments types localhost.json index.js",
     "dev": "yarn rm-dbdata && anvil --config-out localhost.json & make deploy-local",
     "build": "forge build && yarn gen-types",
-    "test": "forge test",
+    "test": "forge clean && forge test",
```

Comment:
The purpose of a build target is to do a singular task, as simply as possible. Do you really need to make this a 'clean and test' build task?

Reply:
The upgreades plugin is looking for a fresh `out` directory everytime.
Will test it once on the build.